### ISSUE TITLE
How can I use swsim on  my sim7600 platform？

### ISSUE THREAD

**Issue by zhuzcpzcp** [tomasz-lisowski/swsim#16]
How can I use swsim on  my  development board such as sim7600 platform ？

**Comment by enginestein** [comment]
Is it possible? I don't think so because swsim is only software based and it is basically simulating a virtual sim card so I dont think that can happen but to achieve what you're saying, maybe you should use arduino with any sim card module. there is a lot of code lying around in the internet regarding that.

**Comment by tomasz-lisowski** [comment]
It is possible but you would need to use something like [SIMtrace2](https://shop.sysmocom.de/SIMtrace2-Hardware-Kit/simtrace2-kit) (running on the [cardem](https://osmocom.org/projects/simtrace2/wiki/Cardem) firmware) to forward the swSIM APDUs back and forth between the dev board and a PC running swSIM and [swICC IFD driver](https://github.com/tomasz-lisowski/swicc-drv-ifd) so it's available via PC/SC.

If you want to do this without any extra hardware, then there is no way to do that right now. Sorry :/

**Comment by tomasz-lisowski** [comment]
@enginestein what do you mean exactly? I believe the OP wants a solution where they don't need to use commercial or even research SIM cards. If they did, they could just slot a real card into their dev board.

I am curious though, please elaborate :>.

**Comment by enginestein** [comment]
> @enginestein what do you mean exactly? I believe the OP wants a solution where they don't need to use commercial or even research SIM cards. If they did, they could just slot a real card into their dev board.
> 
> I am curious though, please elaborate :>.

Actually I don't know much about SIM technology, I am learning from various resources and I said what I thought. Of course I might be wrong. I've been searching open source repositories which contain some material regarding the SIM card tech and I found yours, I ran it and observed it. Well, I thought something stupid at that time which made no sense. I thought that swsim is only software based, so it will not support hardware or any other microcontroller board...well now I think it can.
